dhall/tests/etags/all.tags

```diff
@@ -0,0 +1,61 @@
+
+./tests/etags/let.dhall,66
```

Comment:
So with the tags file in this position, it looks like we'd jump to the wrong path for e.g. `foo`. I wonder whether we should restrict the output file to be in the current directory, so we don't have to adjust the relative paths.

Reply:
All paths are related to directory where `dhall tags` was launched.
I can try to change it to have all paths related to output directory.
Maybe it will be better.
Most probably, use case for this tool is to launch it from root project directory and to generate tags file in root project directory.
But yes, to generate paths related to output directory can be good idea. But it will make code more complex. :)

Reply:
> But yes, to generate paths related to output directory can be good idea. But it will make code more complex. :)

We should avoid that unless explicitly requested. Let's change


    Options.Applicative.metavar "OUTPUT"

to

    Options.Applicative.metavar "FILENAME"

for now.